papis_zotero/sql.py

```diff
@@ -301,17 +288,10 @@ def add_from_sql(input_path: str, output_path: Optional[str] = None) -> None:
             ref = papis.bibtex.create_reference(item)
 
         item["ref"] = ref
```

Comment:
You should be able to remove the `ref` handling in here too, since `papis.commands.add.run` handles that.

Reply:
That's a good idea. I'll integrate the change in the PR!